src/main/java/ea/maps/MapFactory.java

```diff
@@ -1,5 +1,22 @@
 package ea.maps;
 
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
 class MapFactory {
 
+    static List<City> getCityListFromJson(String jsonFilePath)
+        throws IOException {
```

Comment:
Handle this exception in this class and throw a RuntimeException instead so we don't have to handle it in the callee code.